distributed/protocol/core.py

```diff
@@ -81,7 +81,7 @@ def dumps(msg, serializers=None, on_error="message", context=None):
         for i, frame in enumerate(out_frames):
             if type(frame) is memoryview and frame.strides != (1,):
                 try:
-                    frame = frame.cast("b")
```

Comment:
I am unfamiliar here, what is the difference between "b" and "B".  "B" is unsigned bytes/unint8, correct ?

Reply:
Yeah `"b"` is `int8` whereas `"B"` is `uint8`. In general lower case is signed and upper case is unsigned.

Reply:
Ah, right.  That make sense.  Thanks @jakirkham